src/chrome/content/rules/Torproject.xml

```diff
@@ -2,17 +2,10 @@
 
 	<target host="torproject.org" />
 	<target host="*.torproject.org" />
```

Comment:
Ah since I didn't add this wildcard from the original ruleset I missed this.  Added the three test urls.